CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+compliance_tool: Fail to check aasx package without thumbnail
+adapter.aasx: `Property.value` `0` are converted into `NoneType`
+```
+
+As you can see, code-highlighting is used in order to highlight class names or types.
+
+In the issue message, use full text or bullet points to describe your issue in detail.
+Please include a short paragraph on each the
+
+- Expected behavior:  A description of what the expected behavior should be, so that
+  maintainers can understand how the issue differs from the intended functionality.
+- Actual behavior: A description of what is actually happening, which can help pinpoint
+  the cause of the issue.
+- Environment: Information about the operating system, the SDK version used, the version
+  of the specification used, or other relevant technical details that may be
+  contributing to the issue.
+
+Additionally, if you have ideas for how to address the issue, you can include them here.
+
+## Contribute Changes
+
+Here's the standard workflow to contribute changes to Eclipse-BaSyx-Python.
+
+Before contributing, please make sure, you fill out
+the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php). This
+is done by creating an Eclipse account for your git e-mail address and then submitting
+the following
+form: [https://accounts.eclipse.org/user/eca](https://accounts.eclipse.org/user/eca).
+The E-Mail address used to sign the ECA is the same one that needs to be used for
+committing.
+
+After this, the workflow to submit contributions to Eclipse-BaSyx-Python is pretty
+standard, as the picture (based
+on [this blog-post by Tomas Beuzen](https://www.tomasbeuzen.com/post/git-fork-branch-pull/))
+below shows:
+
+![CONTRIBUTING_Workflow](./etc/CONTRIBUTING_Workflow.png)
+
+1. Fork the Eclipse-BaSyx Repository
+2. Clone your fork to your development machine and add Eclipse-BaSyx as `upstream`:
+
+```bash
+git remote add upstream https://github.com/eclipse-basyx/basyx-python-sdk
+```
+
+3. Pull the branch you want to contribute to:
+
+```bash
+git pull upstream <branch_name>
+```
+
+Now, you can create a new local branch in which you can create your changes and actually
+do your changes. When you're done with that, continue with:
+
+4. Push the new branch to your fork:
+
+```bash
+git push origin <your_new_branch>
+```
+
+5. Create a [Pull Request](https://github.com/eclipse-basyx/basyx-python-sdk/pulls) from
+   your fork `<your_new_branch>` to the Eclipse-BaSyx-Python `<branch_name>`
+
+The Eclipse-BaSyx-Python maintainers will then review the pull request and communicate
+the further steps via the comments.
+
+## Commit and Pull Request Messages
+
+In order to effectively communicate, there's some conventions to respect when writing
+commit messages and pull requests.
+
+Similarily, to when creating an issue, the commit title, as well as the PR title should
+be as short as possible, ideally 72 characters or less. In order to achieve this, use

```

Comment:
Since this is copy pasted from the Issue guidelines, the same comments apply here. Maybe we can also find a way not to repeat ourselves, e.g. convert this to bullet points or something else.